tests/other_test.go

```diff
@@ -14,14 +14,14 @@ func TestRunBasicExample(t *testing.T) {
 	t.Parallel()
 
 	options := testhelper.TestOptionsDefault(&testhelper.TestOptions{
-		Testing:      t,
-		TerraformDir: basicExampleTerraformDir,
-		Prefix:       "s2s-basic",
-		TerraformVars: map[string]interface{}{
-			"prefix": options.Prefix,
-		},
+		Testing:       t,
+		TerraformDir:  basicExampleTerraformDir,
+		Prefix:        "s2s-basic",
+		TerraformVars: map[string]interface{}{},
```

Comment:
Why keep `TerraformVars` here is we are not passing any to it?

Reply:
moved out of options